src/model/block/IBlockConfig.ts

```diff
@@ -1,35 +1,14 @@
-export interface IBlockConfig {}
-
 export interface SetContactProperty {
```

Comment:
Why was this not extract along with ISetContactPropertyBlockConfig to either the same file, or it's own file?